core/src/main/kotlin/org/evomaster/core/problem/externalservice/httpws/param/HttpWsResponseParam.kt

```diff
@@ -54,7 +54,16 @@ class HttpWsResponseParam (
     /**
      * @return HTTP status code
      */
-    fun getHttpStatusCode() : Int = status.values[status.index]
+    fun getHttpStatusCode() : Int {
+        if (hasBody()) {
```

Comment:
hmmmm
plenty of cases in which we might want to have body outside a 200.
maybe best to have method that check for body, and return null/false when code is 204